lib/helpers/global-context-helper.js

```diff
@@ -0,0 +1,42 @@
+var isInitialised = false;
+var contextStore = {};
+
+// adopted from: https://github.com/Steveorevo/node-red-contrib-actionflows/pull/20
+module.exports = function (RED) {
+  return {
+    /**
+     * Initialise the runtime global context functions and prepare access to global context via `contextStore`.
+     * If a context store is not provided, an internal store will be created instead to keep things running.
+     * @param {Object} [store] (optional) pass in the context store. Send `null` to use internal memory.
+     */
+    init(store) {
+      if (!isInitialised) {
+        if (store) {
+          contextStore = store;
```

Comment:
Thanks @OidaTiftla very much for the PR, i'm testing locally and it works.

This global `isInitialised` guarantees that the init it's called once, not overwriting anything, right?

Also I was thinking if it's the case here to make a shallow copy of the store `contextStore = { ...store }` during the initialize to avoid side effetcs

Reply:
Thank you @guidone for reviewing the PR.

> This global `isInitialised` guarantees that the init it's called once, not overwriting anything, right?

Yes, exactly.

> Also I was thinking if it's the case here to make a shallow copy of the store `contextStore = { ...store }` during the initialize to avoid side effetcs

It seems reasonable to me, but I'm not a multi-threading expert in node :sweat_smile:

I'm open to all suggestions and ideas. You can change anything you'd like.

```suggestion
          contextStore = { ...store };
```